src/sagemaker/model.py

```diff
@@ -629,8 +629,7 @@ def _compilation_job_config(
         }
 
         if (
-            framework.lower() == "pytorch"
-            or framework.lower() == "tensorflow"
+            (framework.lower() == "pytorch" or framework.lower() == "tensorflow")
```

Comment:
can you please create a framework list  and check if framework is present in it.

```
framework_list = ["pytorch", "tensorflow"]
if framework.lower() in framework_list
```